apps/site/pages/docs/customizing/components.mdx

```diff
@@ -163,6 +173,93 @@ To ensure visual consistency with your store's theme, we highly recommend utiliz
 
 ---
 
+## Importing FastStore UI component styles
```

Comment:
We should remove this whole section since we move it to[ another page](https://faststore-site-git-docs-update-import-styles-faststore.vercel.app/docs/customizing/importing-styles-to-faststore-ui-components#before-you-begin).